### DISCUSSION TITLE
analyzer dep to 5.0.0

### DISCUSSION THREAD

**Discussion by kevinelliott** [dart-backend/angel#80] [General]
https://github.com/dukefirehawk/angel/blob/41167e01bd9fadad57dcfe37d802269f45a32b23/packages/serialize/angel_serialize_generator/pubspec.yaml#L9

Any plan to bump this to 5.0.0?

**Comment by dukefirehawk** [comment]
Done.

**Comment by kevinelliott** [comment]
Thank you!
